demisto_sdk/commands/common/logger.py

```diff
@@ -199,6 +324,22 @@ def replace_escapes(self, message):
 
     set_demisto_logger(demisto_logger)
 
+    if notify_log_file_path and not current_log_file_path_notified:
+        import sys
```

Comment:
Why shouldn't we log this  anyway?

Reply:
[Done](https://github.com/demisto/demisto-sdk/pull/2994/commits/8c7e4b104f8fe9ec88c44caa46967884adfb93c8).